include/llbuild/BuildSystem/CommandResult.h

```diff
@@ -15,6 +15,12 @@
 
 #include <inttypes.h>
 
+#if defined(__linux__) || defined(__GNU__)
+#include <termios.h>
+#else
+#include <unistd.h>
+#endif
```

Comment:
Why is termios needed?

Reply:
I had a build error on Linux which suggeted to import termios for pid_t. If it can be imported with `sys/types.h` on macOS and Linux, we should use this 👍

Reply:
Sounds like a bogus clang diagnostic to me... I don't think of termios.h as the canonical location for pid_t. There should be a cross platform header for this.

My general preference w.r.t. cross platform compatibility is that we avoid using platform-dependent types in the public headers, preferring to hide them in wrapper types with shims inside of Basic/, or making use of the LLVM shims we already have imported (where appropriate).

Reply:
I just added a new header in Basic/ with a definition of llbuild_pid_t which is cross platform in 95bc9a70. Is there any way I can test this on Windows without installing Windows? :)